resources/zib/zib-Alert.xml

```diff
@@ -0,0 +1,203 @@
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-alert-v4.1-2020EN" />
+        <map value="NL-CM:8.3.7" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Flag.status">
+      <path value="Flag.status" />
+      <comment value="Sending systems that don't record an explicit status can use the following guidance to infer a value from the zib:&#xD;&#xA;&#xD;&#xA;* EndDateTime present and in the past: _inactive_&#xD;&#xA;* EndDateTime absent: _active_&#xD;&#xA;* EndDateTime present and in the future - goes against both FHIR and zib definitions, but in a case where this might occur: _active_" />
+      <mapping>
+        <identity value="zib-alert-v4.1-2020EN" />
+        <map value="NL-CM:8.3.5" />
+        <comment value="StartDateTime (implicit, main mapping is on Flag.period.start)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-alert-v4.1-2020EN" />
+        <map value="NL-CM:8.3.8" />
+        <comment value="EndDateTime (implicit, main mapping is on Flag.period.end)" />
+      </mapping>
+    </element>
+    <element id="Flag.category">
+      <path value="Flag.category" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Flag.category:alertName">
+      <path value="Flag.category" />
+      <sliceName value="alertName" />
+      <short value="AlertType" />
+      <definition value="Indicates the type of alert, meaning a rough description of the cause or origin of the warning." />
+      <alias value="AlertType" />
+      <max value="1" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.8.3.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-alert-v4.1-2020EN" />
+        <map value="NL-CM:8.3.6" />
+        <comment value="AlertType" />
+      </mapping>
+    </element>
+    <element id="Flag.code">
+      <path value="Flag.code" />
+      <short value="AlertName" />
+      <definition value="A warning, other than a condition or problem. For example, a patient can be given an ‘Aggressive patient' alert. The warning can be entered in code (there are codes for frequently used alerts), but seeing the dynamic nature of the warnings cf. SARS and Ebola, these alerts will often be entered as free text." />
+      <comment value="One concept from Condition::Concern and AlertName must be selected. If Flag.code contains a code from AlertNaamCodelijst, Flag.extension:concern SHALL not exist. If Flag.extension:concern is filled, Flag.code SHALL contain code 'not-applicable' from system 'http://hl7.org/fhir/ValueSet/data-absent-reason'." />
```

Comment:
This could get some improvements in formatting: `code` instead of Flag.code ect.

Reply:
Deze comment eigenlijk ook op de root zetten ipv op 1 van de 2 concepten die het betreft denk ik